radio/src/gui/480x272/model_setup.cpp

```diff
@@ -96,30 +96,57 @@ enum MenuModelSetupItems {
 #define MODEL_SETUP_SET_FAILSAFE_OFS  100
 #define MODEL_SETUP_SLIDPOT_SPACING   45
 
+#define CURRENT_MODULE_EDITED(k)         (k>=ITEM_MODEL_TRAINER_LABEL ? TRAINER_MODULE : (k>=ITEM_MODEL_EXTERNAL_MODULE_LABEL ? EXTERNAL_MODULE : INTERNAL_MODULE))
+
 #if defined(BINDING_OPTIONS)
 void onBindMenu(const char * result)
 {
+  uint8_t moduleIdx = CURRENT_MODULE_EDITED(menuVerticalPosition);
+
+#if defined(MODULE_D16_EU_ONLY_SUPPORT)
+  if (result == STR_BINDING_25MW_CH1_8_TELEM_OFF) {
+    g_model.moduleData[moduleIdx].pxx.power = R9M_POWER_25;
+    g_model.moduleData[moduleIdx].pxx.receiver_telem_off = true;
+    g_model.moduleData[moduleIdx].pxx.receiver_channel_9_16 = false;
+  }
+  else if (result == STR_BINDING_25MW_CH1_8_TELEM_ON) {
+    g_model.moduleData[moduleIdx].pxx.power = R9M_POWER_25;
+    g_model.moduleData[moduleIdx].pxx.receiver_telem_off = false;
+    g_model.moduleData[moduleIdx].pxx.receiver_channel_9_16 = false;
+  }
+  else if (result == STR_BINDING_500MW_CH1_8_TELEM_OFF) {
+    g_model.moduleData[moduleIdx].pxx.power = R9M_POWER_500;
+    g_model.moduleData[moduleIdx].pxx.receiver_telem_off = true;
+    g_model.moduleData[moduleIdx].pxx.receiver_channel_9_16 = false;
+  }
+  else if (result == STR_BINDING_500MW_CH9_16_TELEM_OFF) {
+    g_model.moduleData[moduleIdx].pxx.power = R9M_POWER_500;
+    g_model.moduleData[moduleIdx].pxx.receiver_telem_off = true;
+    g_model.moduleData[moduleIdx].pxx.receiver_channel_9_16 = true;
+  }
```

Comment:
I know that this is probably what frsky has imlemented but it is horrible that they use different bits for the eu version than for fcc. And more than that, that they reuse the power bits. This looks and feels so wrong ...